src/main/java/com/urswolfer/gerrit/client/rest/http/GerritRestClient.java

```diff
@@ -87,7 +87,7 @@
 
     private static final String JSON_MIME_TYPE = ContentType.APPLICATION_JSON.getMimeType();
     private static final Pattern GERRIT_AUTH_PATTERN = Pattern.compile(".*?xGerritAuth=\"(.+?)\"");
-    private static final int CONNECTION_TIMEOUT_MS = 30000;
+    private static final int CONNECTION_TIMEOUT_MS = 300000; //30000;
```

Comment:
Please revert this.

Reply:
The create project needs more than 1 mins. If revert it then where can we dynamically change it?

Reply:
done